pkg/apis/v1alpha1/descriptors/install.go

```diff
@@ -0,0 +1,22 @@
+package descriptors
+
+import (
+	"github.com/caicloud/nirvana/definition"
+)
+
+var descriptors []definition.Descriptor
+
+func register(ds ...definition.Descriptor) {
+	descriptors = append(descriptors, ds...)
+}
+
+// Descriptor returns a combined descriptor for current version.
+func Descriptor() definition.Descriptor {
+	return definition.Descriptor{
+		Description: "v1alpha1 API",
+		Path:        "/v1alpha1",
+		Consumes:    []string{definition.MIMEJSON},
```

Comment:
unless not all APIs are JSON, those should be added only at `/api` level

Reply:
Add comments to illustrate that other content types can be added if necessary.